tests/ignite/contrib/handlers/test_custom_events.py

```diff
@@ -8,55 +8,58 @@
 
 def test_bad_input():
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations="a")
+    with pytest.warns(DeprecationWarning):
+        with pytest.raises(ValueError):
+            CustomPeriodicEvent(n_iterations="a")
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations=0)
+        with pytest.raises(ValueError):
+            CustomPeriodicEvent(n_iterations=0)
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations=10.0)
+        with pytest.raises(ValueError):
+            CustomPeriodicEvent(n_iterations=10.0)
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_epochs="a")
+        with pytest.raises(ValueError):
+            CustomPeriodicEvent(n_epochs="a")
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_epochs=0)
+        with pytest.raises(ValueError):
+            CustomPeriodicEvent(n_epochs=0)
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_epochs=10.0)
+        with pytest.raises(ValueError):
+            CustomPeriodicEvent(n_epochs=10.0)
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent()
+        with pytest.raises(ValueError):
+            CustomPeriodicEvent()
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations=1, n_epochs=2)
+        with pytest.raises(ValueError):
+            CustomPeriodicEvent(n_iterations=1, n_epochs=2)
 
 
 def test_new_events():
     def update(*args, **kwargs):
         pass
 
-    engine = Engine(update)
-    cpe = CustomPeriodicEvent(n_iterations=5)
-    cpe.attach(engine)
+    with pytest.warns(DeprecationWarning):
```

Comment:
Please, specify warn part of message to match, like above.

Reply:
Done

Reply:
Seems not done, right ?